src/request/change-passphrase/ChangePassphraseApi.js

```diff
@@ -27,40 +26,20 @@ class ChangePassphraseApi extends TopLevelApi { // eslint-disable-line no-unused
     }
 
     /**
+     *
      * @param {KeyguardRequest.SimpleRequest} request
```

Comment:
As we don't really know what comes in here, it would be more correct to use unknown as well.